Dudas Dorian/Lab4/5_3.5.c

```diff
@@ -0,0 +1,105 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
```

Comment:
So the point of all this was to work with functions. You will have to move every operation in a separate function and your main file should only contain stuff like:

```
 addition(B, C, D);
    printf("The new matrix after addition is\n");
    writeMat(D);
```